l10n_it_fattura_accompagnatoria/models/account_invoice.py

```diff
@@ -9,6 +9,18 @@
 class AccountInvoice(models.Model):
 
     _inherit = 'account.invoice'
-    # we need this to be able to call l10n_it_ddt.delivery_data
-    note = fields.Text(
-        'Additional Information', readonly=True, related="comment")
+    # we need these to be able to call l10n_it_ddt.delivery_data
+    note = fields.Text('Notes', readonly=True, related="comment")
+    weight_manual_uom_id = fields.Many2one(
+        'uom.uom', 'Net Weight UoM',
+        default=lambda self: self.env.ref(
+            'uom.product_uom_kgm', raise_if_not_found=False))
+    gross_weight_uom_id = fields.Many2one(
```

Comment:
Questi campi sarebbero da mettere nel modulo `l10n_it_ddt` insieme agli altri: https://github.com/OCA/l10n-italy/blob/b063233bc7a6a7c01211f3f7f040e65cba6c36b8/l10n_it_ddt/models/account.py#L31
e da scrivere in fattura insieme agli altri:
https://github.com/OCA/l10n-italy/blob/b063233bc7a6a7c01211f3f7f040e65cba6c36b8/l10n_it_ddt/models/sale.py#L91
https://github.com/OCA/l10n-italy/blob/b063233bc7a6a7c01211f3f7f040e65cba6c36b8/l10n_it_ddt/models/stock_picking_package_preparation.py#L376

Che ne pensate?

Reply:
> Che ne pensate?

:+1:

Reply:
ciao @eLBati , dopo consulto con @SimoneVagile ho rilasciato le modifiche che dovrebbero venire in contro alla tua richiesta.
potresti controllare e farmi sapere se è ok?
grazie